<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tone_dlg_ui</class>
 <widget class="QDialog" name="Tone_dlg_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tone Dumper</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>144</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bits per sample</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bps_combo_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>8bits</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16bits</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24bits</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32bits</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sample Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sample_rate_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Channels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chan_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Duration (sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="duration_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="freq_edit">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="file_progress">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="file_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="file_btn">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="start_btn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bps_combo_box</tabstop>
  <tabstop>chan_edit</tabstop>
  <tabstop>sample_rate_edit</tabstop>
  <tabstop>duration_edit</tabstop>
  <tabstop>start_btn</tabstop>
  <tabstop>stop_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
